boot/bootstate20.go

```diff
@@ -410,61 +415,51 @@ type bootState20MarkSuccessful struct {
 	bootState20Kernel
 }
 
-// threadBootState20MarkSuccessful is a helper method that will either create a
-// new bootState20MarkSuccessful for the given type, or it will add to the
-// provided bootStateUpdate
-func threadBootState20MarkSuccessful(update bootStateUpdate) (*bootState20MarkSuccessful, error) {
-	var bsmark *bootState20MarkSuccessful
+// chooseBootSnapToMarkSuccessful inspects the specified boot state to pick what
+// boot snap should be marked as successful and use as a valid rollback target.
+// If the
```

Comment:
If the ... ?

Reply:
what you mean you can't read my mind? :smile: